setup.py

```diff
@@ -308,7 +308,7 @@ def run(self):
 extras["torch-speech"] = deps_list("torchaudio") + extras["audio"]
 extras["tf-speech"] = extras["audio"]
 extras["flax-speech"] = extras["audio"]
-extras["vision"] = deps_list("Pillow")
+extras["vision"] = deps_list("Pillow", "opencv-python")
```

Comment:
this will lead to huge dependency update.. however other models do not require cv2. We should just constrain particular place with `requires_backends` instead